src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala

```diff
@@ -79,38 +88,50 @@ class ConsumptionRateFilterStoreImpl(
           settings.setHeaderExtractionEnabled(true)
           settings.detectFormatAutomatically()
           val csvParser = new CsvParser(settings)
-          loadConsumptionRatesFromCSVFor(filePath, csvParser)
+          vehicleType.primaryFuelType == FuelType.Electricity
```

Comment:
what is rational behind this?

Reply:
This was left in during testing...it's a NOP, but I removed it